app/models/ncr/work_order.rb

```diff
@@ -114,13 +114,23 @@ def approving_official
       self.approvers.first
     end
 
-    def current_approver_email_address
+    def current_approver
       if self.pending?
-        self.currently_awaiting_approvers.first.email_address
+        self.currently_awaiting_approvers.first
```

Comment:
In Ruby, I believe it isn't "idiomatic" to use `self` when referring to objects. The only time I usually use self is when defining class methods. Why so much `self` in here?